mempool.go

```diff
@@ -1132,6 +1132,22 @@ func (mp *txMemPool) maybeAcceptTransaction(tx *btcutil.Tx, isNew, rateLimit boo
 		return nil, txRuleError(wire.RejectInsufficientFee, str)
 	}
 
+	// Require that free transactions have sufficient priority to be mined
+	// in the next block.
+	if !cfg.NoRelayPriority && txFee < minFee {
+		txD := &TxDesc{
+			Tx:     tx,
+			Added:  time.Now(),
+			Height: curHeight,
+			Fee:    txFee,
+		}
+		if txD.CurrentPriority(txStore, nextBlockHeight) <= minHighPriority {
+			str := fmt.Sprintf("transaction %v has insufficient "+
```

Comment:
I think the error message should include the current priority it has and the required priority.